### DISCUSSION TITLE
Algorithm Visualizer

### DISCUSSION THREAD

**Discussion by Hardik-Kushwaha** [scaleracademy/open-source-projects#133] [Ideas]
# Algorithm Visualizer

While learning and implementing algorithms as a beginner it's sometimes hard to visualize the control flow of the algorithm 
and my idea is pretty simple, I just want to develop an 'Algorithm Visualizer' that visualize some of the fundamental and widely used algorithms and later on, as an open source project people can contribute algorithms to the project and increase the set of algorithms to the project.

**Comment by BishalBiswal** [comment]
Nice,seems Interesting.

**Comment by sujankumar21** [comment]
This idea is very interesting . As algortihms are quite difficult to understand just by viewing it can be understood in a better way if it is been visualised.so i appreciate this idea.
